generators/pkg/writer/testdata/config.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   steps:
   - args:
     - --context=dir://$(workspaces.input.path)/src
-    - --destination=gcr.io/<use your project>/kaniko-test
+    - --destination=gcr.io/tekton-yolandadu/kaniko-test
```

Comment:
is this intentional?

Reply:
Also, Kaniko usually requires a secret or a service account that has enough credentials to push images. 
I think this is usually already configured for GKE. But other k8s installations might have to configure this

Reply:
It's probably because I paste from another config. Sure I can document it in README.

Reply:
By the way for the service account, can it be the same as the one required for the triggers?